src/components/ProjectCard.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+// import Project from "../components/Project";
+import "./ProjectCard.css";
+
+import { Route, Link } from "react-router-dom";
+
+export default function ProjectCard(props) {
+    const { year, title, img, technologie, bgColor } = props;
+    return (
+        <div
+            style={{ backgroundColor: bgColor }}
+            className="project-card-container"
+        >
+            <div className="project-card-info">
+                <p>{year}</p>
+                <strong>
+                    <h2>{title}</h2>
+                </strong>
+                <p>Full Stack Development</p>
```

Comment:
get this as description from the props

Reply:
but all my projects are the same so what's the need for doing this?

Reply:
just in case you add a project that has different description